CHANGELOG.md

```diff
@@ -13,6 +16,48 @@
 ### Fixed
 
 ### Security
+-->
+
+## [0.18.8] - 2023-04-27
+
+This release includes several dependency updates, some updates to make existing APIs more friendlier and a change to a
```

Comment:
Grammar nit:

```suggestion
This release includes several dependency updates, some updates to make existing APIs friendlier and a change to a
```

or even

```suggestion
This release includes several dependency updates, some updates to make existing APIs more friendly and a change to a
```

Reply:
Yes, of course. At some point I had `more friendly` and wanted to go with just `friendlier`.